tests/DqtApi.Tests/DataverseIntegration/CreateTeacherTests.cs

```diff
@@ -105,8 +122,11 @@ public async Task Given_details_that_does_match_existing_record_does_not_allocat
             Assert.Equal("Notification for QTS Unit Team", crmTask.Subject);
             Assert.Equal(_crmClientFixture.Clock.UtcNow, crmTask.ScheduledEnd);
 
+            var expectedDescription = $"Potential duplicate\nMatched on\n\t- First name: '{firstName}'\n\t- Last name: '{lastName}'\n\t- Date of birth: '{birthDate:dd/MM/yyyy}'\n" +
```

Comment:
Wanna add middle name for completeness?

Will this fall foul of PII/GDPR?

Should it be the `id` instead?

Reply:
> Wanna add middle name for completeness?
Yep good spot.

> Will this fall foul of PII/GDPR?
> Should it be the id instead?
It's ok - the data is already in CRM at this point, this is just to make reviewing the duplicate easier for the end user. This copy is taken from existing file-based implementation.

Reply:
> existing file-based implementation

Seems reasonable, assuming external sentry/3rd party error tracking, has already been evaluated.